docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -153,6 +157,71 @@ The histogram shows historical changes in a particular host's risk score(s). To
 [role="screenshot"]
 image::images/data-tables.png[]
 
+[[troubleshoot-host-risk-score]]
+[discrete]
+=== Troubleshooting
+
+During the installation or upgrade process, you may receive the following error messages:
+
+* `Saved object already exists`
+* `Transform already exists`
+* `Ingest pipeline already exists`
+
+In this case, we recommend that you manually delete the host risk score module, and then re-enable it. To manually delete the module:
+
+. Delete the host risk score saved objects:
+.. From the {kib} main menu, go to **Stack Management** -> **Kibana** -> **Saved Objects**.
+.. Delete the saved objects that have the `Host Risk Score - <space-id>` tag.
++
+[role="screenshot"]
+image::images/delete-hrs-saved-objects.png[Delete host risk score saved objects]
+.. Delete the `Host Risk Score - <space-id>` tag.
++
+[role="screenshot"]
+image::images/delete-hrs-tag.png[Delete host risk score tag]
+. Stop and delete the host risk score transforms. You can do this using the {kib} UI or the {ref}/stop-transform.html[Stop transform API] and {ref}/delete-transform.html[Delete transform API].
+** To delete the host risk score transforms using the {kib} UI:
+.. From the {kib} main menu, go to **Stack Management** -> **Data** -> **Transforms**.
+.. Stop the following transforms, then delete them:
+*** `ml_hostriskscore_latest_transform_<space-id>`
+*** `ml_hostriskscore_pivot_transform_<space-id>`
+** To delete the host risk score transforms using the API, run the following commands in Console:
+.. To stop and delete the latest transform:
```

Comment:
Slight nit -- just removing the `to` because it sounds repetitive since we said it above. 
```suggestion
.. Stop and delete the latest transform:
```